doc/tracking_base_out_vars.csv

```diff
@@ -0,0 +1,3 @@
+Variable Name,Description,Units,Type
+cell,Tracked cell number; generally starts from 1. Untracked cell value can be set; but by default is -1.,n/a,int
+time_cell,Time since cell was first detected.,typically minutes,object/python timedelta
```

Comment:
Isn't that always in minutes, since `transform("min")` is used in `add_cell_time(t)` ?